ui/imports/shared/popups/send/views/NetworksSimpleRoutingView.qml

```diff
@@ -70,18 +72,20 @@ RowLayout {
             Column {
                 id: row
                 spacing: Style.current.padding
+
+                // TODO: This transformation should come from an adaptor outside this component
```

Comment:
any task to link to?

Reply:
Yeah, I have a note to add it in the next `SendModal` improvements cycle.